lib/fastlane/plugin/revenuecat_internal/actions/bump_version_update_changelog_create_pr_action.rb

```diff
@@ -29,6 +29,7 @@ def self.run(params)
         # Ask for new version number
         new_version_number ||= UI.input("New version number: ")
 
+        UI.user_error!("Invalid version number: #{new_version_number}") unless !new_version_number.strip.isEmpty?
```

Comment:
I think something like this? And yeah, maybe writing a test to make sure it fails in this case would be great 😄 
```suggestion
        UI.user_error!("Version number cannot be empty") if new_version_number.blank?
```

Reply:
Blank doesn't exist though?

Reply:
Hmm weird... https://apidock.com/rails/Object/blank%3F it's there... Did you add the `?`?

Reply:
That's in `rails`, but we don't use it?